src/scripts/clipperUI/components/modeButtonSelector.tsx

```diff
@@ -51,25 +49,32 @@ class ModeButtonSelectorClass extends ComponentBase<{}, ClipperStateProp> {
 		let augmentationType: string = AugmentationHelper.getAugmentationType(this.props.clipperState);
 		let augmentationLabel: string = Localization.getLocalizedString("WebClipper.ClipType." + augmentationType + ".Button");
 		let augmentationTooltip = Localization.getLocalizedString("WebClipper.ClipType.Button.Tooltip").replace("{0}", augmentationLabel);
-
-		return <ModeButton imgSrc={ExtensionUtils.getImageResourceUrl(augmentationType + ".png") }
-			label={augmentationLabel} myMode={ClipMode.Augmentation}
-			tabIndex={42} selected={currentMode === ClipMode.Augmentation}
-			onModeSelected={this.onModeSelected.bind(this) }
-			tooltipText={augmentationTooltip}/>;
+		let buttonSelected: boolean = currentMode === ClipMode.Augmentation;
+
+		return {
+			imgSrc: ExtensionUtils.getImageResourceUrl(augmentationType + ".png"),
+			label: augmentationLabel,
+			myMode: ClipMode.Augmentation,
+			selected: buttonSelected,
+			onModeSelected: this.onModeSelected.bind(this),
+			tooltipText: augmentationTooltip
+		};
 	}
 
 	private getFullPageModeButton(currentMode: ClipMode) {
 		if (this.props.clipperState.pageInfo.contentType === OneNoteApi.ContentType.EnhancedUrl) {
 			return undefined;
 		}
-
-		return <ModeButton imgSrc={ExtensionUtils.getImageResourceUrl("fullpage.png")}
-			label={Localization.getLocalizedString("WebClipper.ClipType.ScreenShot.Button")}
-			myMode={ClipMode.FullPage} tabIndex={40}
-			selected={currentMode === ClipMode.FullPage}
-			onModeSelected={this.onModeSelected.bind(this) }
-			tooltipText={Localization.getLocalizedString("WebClipper.ClipType.ScreenShot.Button.Tooltip")}/>;
+		let buttonSelected: boolean = currentMode === ClipMode.FullPage;
```

Comment:
Was there a particular reason as to why buttonSelected was made into its own variable in this and other methods?